### PR TITLE
Add 5.1 release notes for changed `attrs.html` template location

### LINKED ISSUE #10874: Template move missing from 5.1 release notes
### Pertinent section of the Wagtail docs

https://docs.wagtail.org/en/stable/releases/5.1.html#other-features

### Details

The template `wagtailadmin/tables/attrs.html` was moved to `wagtailadmin/shared/attrs.html`

The result of this was to break all the custom cell types in our generic chooser views. A note on this in the release notes could have avoided the lengthy troubleshooting.

### PR DESCRIPTION
**Pull request by lb-**
Fixes #10874

To be backported to 5.1.* branch

### CODE REVIEW

**Review by laymonage — APPROVED** [review]

**Comment by lb-** [comment]
Added to 5.1 stable branch 2f8fe9aa3f8934509f8e211af0c7c3d189397943

File: docs/releases/5.1.md [hunk 1]
```diff
@@ -535,3 +535,12 @@ Additionally, two new events will be dispatched when the dialog visibility chang
 | ------ | ----------------- |
 | Show   | `w-dialog:shown`  |
 | Hide   | `w-dialog:hidden` |
+
+## Shared template `.../tables/attrs.html` has been renamed to `.../shared/attrs.html`
```
Comment (laymonage) [inline docs/releases/5.1.md:539]: Sorry, missed this during review – this should've been heading level 3

Comment (lb-) [reply docs/releases/5.1.md:539]: I'll fix and push to master. Sorry I missed that one also.

Comment (lb-) [reply docs/releases/5.1.md:539]: Fixed aa52f84ec47faa9abf94c11207a7e459fbb3201b

**Pull request merged**